app/models/self_registration/housemate.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+#
+#  Copyright (c) 2023, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+
+class SelfRegistration::Housemate
```

Comment:
Das gehört in den Wagon. Es kann ja dann dort Housemate von MainPerson erben statt umgekehrt